src/main/scala/org/scalajs/cli/Scalajsld.scala

```diff
@@ -79,7 +100,7 @@ object Scalajsld {
         .unbounded()
         .action { (x, c) => c.copy(moduleInitializers = c.moduleInitializers :+ x) }
         .text("Execute the specified main(Array[String]) method on startup")
-      opt[File]('o', "output")
+      opt[OutputFile]('o', "output")
         .valueName("<file>")
```

Comment:
There were two additional configuration options introduced in Scala.js 1.5.0:
- `moduleSplitStyle`
- `outputPatterns`

We might want to add these as well.